src/deriver/ScopedKeys.js

```diff
@@ -87,6 +95,34 @@ class ScopedKeys {
         });
     });
   }
+
+  /**
+   * Derive the special scoped key for legacy sync.
+   * @method deriveLegacySyncKey
```

Comment:
@rfk do you think `* @example` would be useful here? or add a second example above at https://github.com/mozilla/fxa-crypto-relier/pull/14/files#diff-11e7ef22cf89a9330416c33c4641f691R17 to fully document this new behaviour otherwise it is a bit hidden for new devs / contributors.   (Not sure if the doc thing supports dual examples :) )